osu.Game.Tournament/TournamentGameBase.cs

```diff
@@ -141,6 +145,23 @@ private void load(Storage storage, FrameworkConfigManager frameworkConfig)
             });
         }
 
+        private void readStableConfig()
+        {
+            if (storage.Exists(stable_config))
+            {
+                using (Stream stream = storage.GetStream(stable_config, FileAccess.Read, FileMode.Open))
+                using (var sr = new StreamReader(stream))
+                {
+                    stableInfo = JsonConvert.DeserializeObject<StableInfo>(sr.ReadToEnd());
+                }
+            }
+
+            if (stableInfo == null)
+                stableInfo = new StableInfo();
+
+            dependencies.Cache(stableInfo);
+        }
```

Comment:
Not sure I like the fact that the json is being read here, but written in `FileBasedIPC`.